intranet/static/js/eighth/distribute_group.js

```diff
@@ -15,19 +16,19 @@ $(function() {
         var chk = $(this).prop("checked");
         var name = $(this).attr("name");
         var num_checked = 0;
-        var $checkboxes = $(".user-item[name='"+name+"']");
+        var $checkboxes = $(".user-item[name='" + name + "']");
 
         $checkboxes.each(function() {
-            if($(this).prop("checked")) {
+            if ($(this).prop("checked")) {
                 num_checked++;
             }
         });
 
-        $e = $(".select-all[data-name='"+ name + "']");
-        if(num_checked == 0) {
+        $e = $(".select-all[data-name='" + name + "']");
+        if (num_checked === 0) {
             $e.prop("checked", false);
             $e.prop("indeterminate", false);
-        } else if(num_checked == $checkboxes.length) {
+        } else if (num_checked === $checkboxes.length) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'num_checked' is not in camel case.](https://www.codacy.com/app/tjhsst-ion/ion/file/2755452138/issues/source?bid=3212528&fileBranchId=3355681#l31)